index.js

```diff
@@ -69,6 +62,19 @@ db.once('open', async () => {
   } catch (error) {
     console.error(error)
   }
+}
+
+mongoose.connect(process.env.DB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
+
+const db = mongoose.connection;
+
+db.on('error', (err) => {
+  console.error(err);
+});
+
+db.once('open', () => {
+  new CronJob('0 0 12 * * * *', () => putAllMoviePagesInDatabase(), null, true, 'America/Los_Angeles');
+  putAllMoviePagesInDatabase();
```

Comment:
You don't need to call `putAllMoviePagesInDatabase` here. Only inside cron callback.

Reply:
If you want to start cron immediately use `start` method of `cron` - [example](https://github.com/kelektiv/node-cron/blob/master/examples/every_10_minutes.js)